src/arx7.js

```diff
@@ -267,4 +289,14 @@ export class ARX7 {
       log(`ERROR: ${message.command}`);
     }
   }
+
+  moduleList() {
+    let list = '';
+
+    this.commands.forEach(command => {
+      list += command.constructor.name.toLowerCase() + ', ';
+    });
+
+    return list.substring(0, list.length - 2);
```

Comment:
Thanks, was trying to remember what `join` was called the other night and was having trouble :+1: